pkgs/development/libraries/freetype/default.nix

```diff
@@ -8,34 +8,17 @@
 }:
 
 let
-  version = "2.6.2";
-
-  # Don't use fetchpatch. It mangles them. That's an hour I'll never get back.
-  fetchbohoomil = name: sha256: fetchurl {
-    url = https://raw.githubusercontent.com/bohoomil/fontconfig-ultimate/254b688f96d4a37f78fb594303a43160fc15c7cd/freetype/ + name;
-    inherit sha256;
-  };
+  version = "2.6.5";
 in
 with { inherit (stdenv.lib) optional optionals optionalString; };
 stdenv.mkDerivation rec {
   name = "freetype-${version}";
 
   src = fetchurl {
-    url = "mirror://sourceforge/freetype/${name}.tar.bz2";
-    sha256 = "14mqrfgl18q2by1yzv6vcxi97zjy4kppcgsqf312mhfwgkpvvxms";
+    url = "mirror://savannah/freetype/${name}.tar.bz2";
+    sha256 = "1w5c87s4rpx9af5b3mk5cjd1yny3c4dq5p9iv3ixb3vr00a6w2p2";
   };
 
-  patches = []
-    # mingw: these patches use `strcasestr` which isn't available on windows
-    ++ optionals (useEncumberedCode && stdenv.cross.libc or null != "msvcrt" ) [
-      (fetchbohoomil "01-freetype-2.6.2-enable-valid.patch"
```

Comment:
We still want to enable validation and sub-pixel rendering, don't we? Fedora has those patches: http://pkgs.fedoraproject.org/cgit/rpms/freetype.git/tree/